AWSS3/AWSS3TransferUtility.h

```diff
@@ -621,7 +621,7 @@ handleEventsForBackgroundURLSession:(NSString *)identifier
                                                                 _Nullable AWSS3TransferUtilityUploadCompletionHandlerBlock * _Nullable completionHandlerReference))uploadBlocksAssigner
                                 downloadTask:(nullable void (^)(AWSS3TransferUtilityDownloadTask *downloadTask,
                                                                 _Nullable AWSS3TransferUtilityProgressBlock * _Nullable downloadProgressBlockReference,
-                                                                _Nullable AWSS3TransferUtilityDownloadCompletionHandlerBlock * _Nullable completionHandlerReference))downloadBlocksAssigner;
+                                                                _Nullable AWSS3TransferUtilityDownloadCompletionHandlerBlock * _Nullable completionHandlerReference))downloadBlocksAssigner NS_SWIFT_UNAVAILABLE("Use enumerateToAssign(blocks:) instead.");
```

Comment:
This should go in as a breaking change and minor version update

Reply:
That is probably true. With the current version of Xcode these functions fail the build when working with Swift so it is already an external breaking change. Earlier versions of Xcode worked with it. Do you think this requires a minor version bump in this case? Another option would be to mark these methods as deprecated and point to the new public method which does work with Swift and when we are ready to bump the minor version we could add the `NS_SWIFT_UNAVAILABLE` attribute. That would allow existing code to continue working with older versions of Xcode and provide a warning to direct developers to the new method.